BUILD

```
@@ -53,8 +53,8 @@ kt_android_library(
 # This work around is to use an empty java_library to include the cc_library dependencies needed
 # for the kotlin jni layer
 android_library(
-    name = "java_cc_alias",
-    srcs = ["library/kotlin/io/envoyproxy/envoymobile/EnvoyKotlinEmptyClass.java"],
+    name = "envoy_engine",
```

```suggestion
    name = "envoy_engine_lib",
```
due to Bazel naming conventions.